### PR TITLE
fix argocd apps and change labels

### PR DESCRIPTION
**Pull request by rasheedamir**

### CODE REVIEW

File: 01-gabbar/configs/00-preview/space.yaml [hunk 1]
```diff
@@ -3,11 +3,8 @@ kind: Space
 metadata:
   name: gabbar-pr
   labels:
-    team: gabbar
-    kind: pr
-    stakater.com/workload-monitoring: 'true'
```
Comment (hanzala1234) [inline 01-gabbar/configs/00-preview/space.yaml:8]: why are we removing this?

Comment (hanzala1234) [reply 01-gabbar/configs/00-preview/space.yaml:8]: its required for monitoring from workload prometheus

File: 01-gabbar/configs/00-preview/space.yaml [hunk 2]
```diff
@@ -3,11 +3,8 @@ kind: Space
 metadata:
   name: gabbar-pr
   labels:
-    team: gabbar
-    kind: pr
```
Comment (hanzala1234) [inline 01-gabbar/configs/00-preview/space.yaml:7]: this is required as this label is in SD

File: 01-gabbar/configs/00-preview/space.yaml (hunk 1 above)
Comment (rasheedamir) [reply 01-gabbar/configs/00-preview/space.yaml:8]: tenant-operator automatically add those

File: 01-gabbar/configs/00-preview/space.yaml (hunk 2 above)
Comment (rasheedamir) [reply 01-gabbar/configs/00-preview/space.yaml:7]: I want to replace them with env.stakater.com/kind

Comment (hanzala1234) [reply 01-gabbar/configs/00-preview/space.yaml:7]: make sense. need to change in stakater gitops first.

File: 01-gabbar/configs/00-preview/space.yaml (hunk 1 above)
Comment (hanzala1234) [reply 01-gabbar/configs/00-preview/space.yaml:8]: make sense

File: 01-gabbar/configs/00-preview/space.yaml (hunk 2 above)
Comment (rasheedamir) [reply 01-gabbar/configs/00-preview/space.yaml:7]: for now just add it as an extra one

**Review by hanzala1234 — DISMISSED** [review]

**Review by hanzala1234 — APPROVED** [review]

**Pull request merged**